code/__defines/mob_stats.dm

```diff
@@ -5,3 +5,14 @@
 #define STAT_BIO						"Biology"
 #define STAT_ROB						"Robustness"
 #define STAT_TGH						"Toughness"
+
+#define ALL_STATS	list(STAT_MEC,STAT_COG,STAT_BIO,STAT_ROB,STAT_TGH)
```

Comment:
It's better to use something like `subtypesof(/datum/stat)` to get all stats.
That list can become out of date.

Reply:
```
var/list/all_stats = list()
for(var/type in subtypesof(/datum/stat) - /datum/stat)
    var/datum/stat/S = type
    all_stats += initial(S.name)
```

Reply:
Can you do it yourself after merging, i dont know where i should put it.